File: examples/prayer/times/go/prayerTimes.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/prayer"
)

// Get the prayer (salah) times for a location on a given date
func PrayerTimes() {
	prayerService := prayer.NewPrayerService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := prayerService.Times(&prayer.TimesRequest{
		Location: "london",
	})
	fmt.Println(rsp, err)
}

File: examples/prayer/times/node/prayerTimes.js
const { PrayerService } = require("m3o/prayer");

// Get the prayer (salah) times for a location on a given date
async function prayerTimes() {
  let prayerService = new PrayerService(process.env.MICRO_API_TOKEN);
  let rsp = await prayerService.times({
    location: "london",
  });
  console.log(rsp);
}

prayerTimes();
